components/Dialog/Dialog.tsx

```diff
@@ -77,13 +91,9 @@ export const DialogContent = React.forwardRef<
 >(({ children, ...props }, forwardedRef) => (
   <StyledContent {...props} ref={forwardedRef}>
     {children}
-    <StyledCloseButton asChild>
-      <IconButton css={{ color: '$hiContrast' }}>
-        <Cross1Icon />
-      </IconButton>
-    </StyledCloseButton>
+    <DialogCloseIconButton />
   </StyledContent>
 ));
 
 export const DialogTrigger = DialogPrimitive.Trigger;
-export const DialogClose = StyledCloseButton;
+export const DialogClose = DialogPrimitive.Close;
```

Comment:
I'm wondering if it makes sense to keep `DialogCloseIconButton` named `DialogClose` and call the primitive `DialogClosePrimitive`, wdyt?

As the `DialogCloseIconButton` might be used more often, I think it would be nice to have a simpler/shorter name. I agree that `DialogCloseIconButton` says exactly what the component is at a glance, so feel free to keep it if you prefer it.

Reply:
@paulocfjunior What about having clearer naming rather than misleading names, even if we drop `DialogClose` ?

- I think your proposal of `DIalogClosePrimitive` is clearer than `DialogClose`, thus I'd be in favor of it
- I think `DialogCloseIconButton` is also the clearest naming possible